XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -1011,8 +1017,9 @@ public virtual void ProcessTransformData()
             var lastHitObject = PrimaryPointer.Result.LastHitObject;
 
             var scale = manipulationTarget.localScale;
-            var scaledSize = Collider.size * scale.y;
-            var scaledCenter = Collider.center * scale.y;
+            var bounds = Collider.bounds;
+            var scaledSize = bounds.size * scale.y;
```

Comment:
I'm a little nervous about this change.
Sometimes the collider size and center is different from the collider size and center.